.github/workflows/functional_tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Run functional tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  DIKTAT_VERSION: 0.1.0
+  KTLINT_VERSION: 0.37.1
+
+jobs:
+  run-functional-tests:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run diKTat from cli
+        continue-on-error: true
+        run: |
+          cat pom.xml | grep "<diktat-check.version>.*</diktat-check.version>" | head -1 |awk -F'[><]' '{ENVIRON[DIKTAT_VERSION]=$3 ; print ENVIRON[DIKTAT_VERSION]}'
+          cat pom.xml | grep "<ktlint.version>.*</ktlint.version>" | head -1 |awk -F'[><]' '{ENVIRON[KTLINE_VERSION]=$3 ; print ENVIRON[KTLINE_VERSION]}'
+          curl -sSLO https://github.com/pinterest/ktlint/releases/download/$KTLINT_VERSION/ktlint && chmod a+x ktlint
+          curl -sSLO https://github.com/cqfn/diKTat/releases/download/v$DIKTAT_VERSION/diktat-$DIKTAT_VERSION.jar
+          ./ktlint -R diktat-$DIKTAT_VERSION.jar "diktat-rules/src/test/kotlin/**/FunctionalTestFile.kt" > out
+          
+      - name: Check output
+        id: cli-check
+        run: |
+          cat out
+          grep -E "[VARIABLE_NAME_INCORRECT_FORMAT]" out
+        
+      - name: Run diKTat from gradle
+        continue-on-error: true
+        run: gradle diktatCheck > out 2> err
+        
+      - name: Check gradle
+        run: |
+          cat out
+          if [ -z "`grep -E "[VARIABLE_NAME_INCORRECT_FORMAT]" out`" ]; then exit 1; fi
```

Comment:
Why are these checks different from the first one (just `grep -E`)? Also, for me this variant seems more readable, with explicit return code.